src/k8s_etcd.rs

```diff
@@ -176,6 +176,36 @@ impl InMemoryK8sEtcd {
         self.deleted_keys.lock().await.insert(key.to_string());
         Ok(())
     }
+
+    pub(crate) async fn update_member(&self, value: String) -> Result<()> {
+        let etcd_client = self.etcd_client.clone().context("etcd client not configured")?;
```

Comment:
needed to set clone as without clone it was failing 
` move occurs because `self.etcd_client` has type `std::option::Option<Arc<Client>>`, which does not implement the `Copy` trait`

Reply:
We can just borrow the Arc instead of cloning the option

```suggestion
        let etcd_client = self.etcd_client.as_ref().context("etcd client not configured")?;
```

This should work as well